server/channels/app/migrations.go

```diff
@@ -559,6 +559,18 @@ func (s *Server) doPostPriorityConfigDefaultTrueMigration() {
 	}
 }
 
+func (s *Server) doElasticsearchFixChannelIndex() {
+	// If the migration is already marked as completed, don't do it again.
+	if _, err := s.Store().System().GetByName(model.MigrationKeyElasticsearchFixChannelIndex); err == nil {
+		return
+	}
+
+	if _, appErr := s.Jobs.CreateJob(model.JobTypeElasticsearchFixChannelIndex, nil); appErr != nil {
+		mlog.Error("failed to start job for fixing Elasticsearch channels index", mlog.Err(appErr))
```

Comment:
Let's use `mlog.Fatal` here to remain consistent with other migrations.

Reply:
Done.